src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -155,6 +148,23 @@ private string CreateJsonWebToken(JObject payload, SigningCredentials signingCre
                 JsonWebTokenManager.KeyToHeaderCache.TryAdd(JsonWebTokenManager.GetHeaderCacheKey(signingCredentials), rawHeader);
             }
 
+            var expires = JwtTokenUtilities.GetDateTime(JwtRegisteredClaimNames.Exp, payload);
```

Comment:
We don't care about the value, just did the user set it.
We could write: 
var expires = payload.TryGetValue(JwtRegisteredClaimNames.Exp, out JToken token);

This returns a bool.

Reply:
Also, if we truly don't care about the value then I would recommend using a [Discard](https://docs.microsoft.com/en-us/dotnet/csharp/discards). For example:
var expires = payload.TryGetValue(JwtRegisteredClaimNames.Exp, out _);

Reply:
This is a really good suggestion, thanks!  :) 
I've made the change.